subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolutionstrategy/DefaultResolutionStrategy.java

```diff
@@ -324,6 +335,7 @@ public DefaultResolutionStrategy copy() {
         if (!isDependencyVerificationEnabled()) {
             out.disableDependencyVerification();
         }
+        out.getUseGlobalDependencySubstitutionRules().convention(useGlobalDependencySubstitutionRules.get());
```

Comment:
This does `get()` on purpose because this is a copy and the two _properties_ should **not** be linked.

Reply:
Might be good to have a test confirming that

Reply:
Good point. I added a unit test.